File: frontend/src/app/dto/RequestLots.ts
import {Pageable} from './Pageable';
import {Filter} from './Filter';

export class RequestLots {
  public pageable: Pageable;
  public filter: Filter;

  constructor(pageable: Pageable, filter: Filter) {
    this.pageable = pageable;
    this.filter = filter;
  }
}

File: src/main/java/coursework/prostranstvo/Model/DTO/RequestLots.java
package coursework.prostranstvo.Model.DTO;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class RequestLots {
    private Pageable pageable;
    private Filter filter;
}
